lib/mayaUsd/render/vp2RenderDelegate/proxyRenderDelegate.cpp

```diff
@@ -332,6 +345,25 @@ void SelectionChangedCB(void* data)
 }
 #endif
 
+#ifdef MAYA_HAS_DISPLAY_LAYER_API
+void DisplayLayerMembershipChangedCB(void* data)
```

Comment:
According to our coding standards, "Function/Method Names All functions should be lowerCamelCase."